### ISSUE TITLE
false negative: no-internal-module

### ISSUE THREAD

**Issue by unional** [palantir/tslint#1069]
https://github.com/typed-typings/npm-chai/blob/master/lib/Chai.d.ts#L28-L33
```ts
/* tslint:disable:no-internal-module */
declare global {
  interface Object {
    should: A.Assertion;
  }
}
```
Global augmentation is detected as internal module.

**Comment by JKillian** [comment]
For context, this is a fairly new feature of TypeScript: https://github.com/Microsoft/TypeScript/pull/6213
